src/utils/enums.ts

```diff
@@ -0,0 +1,4 @@
+export enum ViewMode {
+  FULL = 'FULL',
+  TEASER = 'TEASER',
+}
```

Comment:
As discussed in gather,  we might need to refactor the way the nodes and blocks work with viewMode, lets discuss with Lynn before implementing these changes